webapp-django/crashstats/crashstats/templates/crashstats/partials/correlations.html

```diff
@@ -1,82 +1,83 @@
+                    </p>
+                </div>
+            </li>
+            <li>
+                <h3 class="modver-heading heading" role="tab" aria-expanded="false" aria-selected="false">
+                    <a href="#interesting-modules-with-versions-correlation{{ loop.index }}">
+                        <span aria-hidden="true" data-icon="&#xe803"></span>Modules By Versions
+                    </a>
+                </h3>
+                <div id="interesting-modules-with-versions-correlation{{ loop.index }}" class="content-pane" role="tabpanel" aria-labelledby="modver-heading" tabindex="-1">
+                    <button name="interesting-modules-with-versions" class="load-version-data">Load</button>
+                </div>
+            </li>
+            <li class="add">
+                <h3 class="addons-heading heading" role="tab" aria-expanded="false" aria-selected="false">
+                    <a href="#interesting-addons-correlation{{ loop.index }}">
+                        <span aria-hidden="true" data-icon="&#xe803"></span>Add-Ons
+                    </a>
+                </h3>
+                <div id="interesting-addons-correlation{{ loop.index }}" class="content-pane" role="tabpanel" aria-labelledby="addons-heading" tabindex="-1">
+                    <p class="spinner">Loading
+                        <img src="{{ static('img/loading.png') }}" width="16" height="17" alt="loading spinner" />
+                    </p>
+                </div>
+            </li>
+            <li>
+                <h3 class="addonsbyver-heading heading" role="tab" aria-expanded="false" aria-selected="false">
+                    <a href="#interesting-addons-with-versions-correlation{{ loop.index }}">
+                        <span aria-hidden="true" data-icon="&#xe803"></span>Add-Ons By Version
+                    </a>
+                </h3>
+                <div id="interesting-addons-with-versions-correlation{{ loop.index }}" class="content-pane" role="tabpanel" aria-labelledby="addonsbyver-heading" tabindex="-1">
+                    <button name="interesting-addons-with-versions" class="load-version-data">Load</button>
+                </div>
+            </li>
+            <li>
+                <h3 class="cpucores-heading heading" role="tab" aria-expanded="false" aria-selected="false">
+                    <a href="#core-counts-correlation{{ loop.index }}">
+                        <span aria-hidden="true" data-icon="&#xe803"></span>CPU Cores
+                    </a>
+                </h3>
+                <div id="core-counts-correlation{{ loop.index }}" class="content-pane" role="tabpanel" aria-labelledby="cpucores-heading" tabindex="-1">
+                    <p class="spinner">Loading
+                        <img src="{{ static('img/loading.png') }}" width="16" height="17" alt="loading spinner" />
+                    </p>
+                </div>
+            </li>
+        </ul>
+    </div>
```

Comment:
The gist of this big block of change is that it used to be things like:
```
<div id="something">...</div>
```
Now that I wrap it in a loop you can't repeat the same ID so I basically changed it to:
```
{% for thing in things %}
<div id="something{{ loop.index }}">...</div>
{% endfor %}
```
so that the IDs became unique.